tomlparams/utils.py

```diff
@@ -34,3 +34,23 @@ def load_toml(path):
                 f'\n***\n*** Problem parsing {path}:\n***\n', file=sys.stderr
             )
             raise
+
+
+def concatenate_keys(d: dict, sep='.'):
```

Comment:
Mark return type here? Think it would be `Generator[tuple[str, Any], None, None]`. Or could be more specific and replace `Any` with supported types in TOML.

Hmm, actually - do we know that the keys will always be strings? Is that a TOML requirement?